### ISSUE TITLE
AS can't import common module

### ISSUE THREAD

**Issue by DmyMi** [icerockdev/moko-resources#110]
After upgrading moko-resources to 0.13.0 and kotlin 1.4.0 i get this error when doing gradle sync in AS:
```
Warning:<i><b>project ':presentation': Unable to build Kotlin project configuration</b>
Details: java.lang.reflect.InvocationTargetException: null
Caused by: java.lang.IllegalStateException: Couldn't resolve metadata artifact for ModuleDependencyIdentifier(groupId=dev.icerock.moko, moduleId=parcelize) in configuration ':presentation:iosArm64CompileKlibraries'</i>
```

And Android Studio doesn't import the common&ios parts of the module. so i loose all ide features for imports, completion, etc. for that module :( everything works fine when i remove the moko-resources dependency.

module's build.gradle.kts:
```
import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget

plugins {
    id("com.android.library")
    kotlin("multiplatform")
    kotlin("kapt")
    kotlin("android.extensions")
    id("dev.icerock.mobile.multiplatform-resources")
}

android {
    compileSdkVersion(Deps.Versions.compileSdk)
    buildToolsVersion(Deps.Versions.buildTools)

    defaultConfig {
        minSdkVersion(Deps.Versions.minSdk)
        targetSdkVersion(Deps.Versions.compileSdk)
        versionCode = 1
        versionName = "1.0"
        testInstrumentationRunner = "android.support.test.runner.AndroidJUnitRunner"
    }

    compileOptions {
        sourceCompatibility = JavaVersion.VERSION_1_8
        targetCompatibility = JavaVersion.VERSION_1_8
    }

    sourceSets {
        getByName("main").java.srcDirs("src/androidMain/kotlin")
        getByName("main").res.srcDirs("src/androidMain/res")
        getByName("main").manifest.srcFile("src/androidMain/AndroidManifest.xml")
        getByName("androidTest").java.srcDirs("src/androidTest/kotlin")
    }
}

version = "1.0"

kotlin {
    ios() {
        binaries {
            framework {
                export(Deps.Kotlin.Multiplatform.napier)
            }
        }
    }

    android()

    sourceSets {
        val commonMain by getting {
            dependencies {
                // other deps
                api(Deps.Kotlin.Multiplatform.resources)
            }
        }
        val commonTest by getting {
            dependencies {
                // other deps
            }
        }
        val androidMain by getting {
            dependencies {
                // other deps
            }
        }
        val androidTest by getting {
            dependencies {
                // other deps
            }
        }

        val iosMain by getting {
            dependencies {
               // other deps
            }
        }
        val iosTest by getting
    }
}

multiplatformResources {
    multiplatformResourcesPackage = "my.package.presentation"
}
```

in gradle.properties i have:
```
kotlin.mpp.enableGranularSourceSetsMetadata=true
kotlin.native.enableDependencyPropagation=false
xcodeproj=ios/app.xcworkspace
```

**Comment by Alex009** [comment]
hi, @DmyMi !
@Skeptick  is right, on kotlin 1.4.0 now bug, which should be fixed in 1.4.10
https://youtrack.jetbrains.com/issue/KT-41083#focus=Comments-27-4365126.0-0

**Comment by DmyMi** [comment]
thank you @Skeptick  @Alex009 ! it works
